enaml/qt/docking/q_icon_widget.py

```diff
@@ -93,6 +94,8 @@ def minimumSizeHint(self):
         size = self._icon_size
         if not size.isValid():
             size = QSize(16, 16)
+        if IS_QT6:
```

Comment:
Could you add a comment on this one ?

Reply:
The function getContentsMargins is obsolete and https://doc.qt.io/qt-5/qwidget-obsolete.html 
it seems it has been removed.

Reply:
Updated to use `contentsMargins()` now